src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/TransformationToolbar.java

```diff
@@ -31,89 +31,52 @@
+        + testId));
+    Assert.assertTrue(Helper.isElementExists(Helper.getCssSelectorByDataTestId("toolbar-icon-"
+        + testId)));
   }
   @Then("Click on the function names toggle with testId as \\\"(.*)\\\" and \\\"(.*)\\\"")
   public void clickOnTheSliderButton(String testId, String iconLabelName) {
-    try {
-      Helper.waitSeconds(5);
-      WaitHelper.waitForElementToBeEnabled
-              (Helper.locateElementByTestId("transformations-toolbar-icons-function-name-toggler"));
-      WebElement toggle = Helper.locateElementByTestId("transformations-toolbar-icons-function-name-toggler");
-      WaitHelper.waitForElementToBeDisplayed(toggle);
-      WebElement ele = Helper.locateElementByXPath
-        ("//input[@data-testid='transformations-toolbar-icons-function-name-toggler']");
-      Actions action = new Actions(SeleniumDriver.getDriver());
-      WaitHelper.waitForPageToLoad();
-      action.moveToElement(ele).perform();
-      ele.click();
-      WaitHelper.waitForPageToLoad();
-      WebElement icon = Helper.locateElementByTestId("toolbar-icon-title-" + testId);
-      String text = icon.getText();
-      String actual = iconLabelName;
-      Assert.assertEquals(text, actual);
-    } catch (Exception e) {
-      System.err.println("error: " + e);
-    }
+    Helper.waitSeconds(5);
+    WaitHelper.waitForElementToBeEnabled
+    (Helper.locateElementByTestId("transformations-toolbar-icons-function-name-toggler"));
+    WebElement toggle = Helper.locateElementByTestId("transformations-toolbar-icons-function-name-toggler");
+    WaitHelper.waitForElementToBeDisplayed(toggle);
+    WebElement ele = Helper.locateElementByXPath
+    ("//input[@data-testid='transformations-toolbar-icons-function-name-toggler']");
+    Actions action = new Actions(SeleniumDriver.getDriver());
+    WaitHelper.waitForPageToLoad();
+    action.moveToElement(ele).perform();
+    ele.click();
+    WaitHelper.waitForPageToLoad();
+    WebElement icon = Helper.locateElementByTestId("toolbar-icon-title-" + testId);
+    String text = icon.getText();
+    String actual = iconLabelName;
+    Assert.assertEquals(text, actual);
   }
   @Then("Verify by Clicking on the Up and Down arrow icon")
   public void clickOnTheUpAndDownArrowButton() {
-    try {
-      WaitHelper.waitForPageToLoad();
-      WaitHelper.waitForElementToBeDisplayed(Helper.locateElementByTestId("toolbar-header-toggler"));
-      WebElement ele = SeleniumDriver.getDriver().findElement(
-              By.xpath("//*[@data-testid='toolbar-header-toggler']"));
-      JavascriptExecutor executor = (JavascriptExecutor) SeleniumDriver.getDriver();
-      executor.executeScript("arguments[0].click();", ele);
-    } catch (Exception e) {
-      System.err.println("error:" + e);
-    }
+    WaitHelper.waitForPageToLoad();
+    WaitHelper.waitForElementToBeDisplayed(Helper.locateElementByTestId("toolbar-header-toggler"));
+    WebElement ele = SeleniumDriver.getDriver().findElement(
```

Comment:
Same as above, use `Helper.locateElementByTestId()`, and `ElementHelper.ClickOnElement()`

Reply:
Updated the code with `Helper.locateElementByXPath()`